test/test_new_command.rb

```diff
@@ -89,9 +100,11 @@ def site_template
     end
 
     should "force created folder" do
-      capture_stdout { Jekyll::Commands::New.process(@args) }
-      output = capture_stdout { Jekyll::Commands::New.process(@args, "--force") }
-      assert_match(%r!New jekyll site installed in!, output)
+      capture_stdout { Jekyll::Commands::New.process(@args, "--force") }
+      output = Jekyll.logger.messages.last
+      success_message = "New jekyll site #{@title.cyan} " \
+      "installed in #{@full_path.cyan}."
+      assert_includes output, success_message
```

Comment:
Since we are modifying an existing test here, rather than creating a new one, it makes me uncomfortable to go from `assert_match` to `assert_includes`, but I guess the old test wasn't even checking for the path, so it's not as if the new test is any more lenient.

:+1: